.github/workflows/build.yml

```diff
@@ -180,7 +180,8 @@ jobs:
           export EXTERNAL_S3_BUCKET_AWS_ACCESS_KEY_ID="${{secrets.EXTERNAL_S3_BUCKET_AWS_ACCESS_KEY_ID}}"
           export EXTERNAL_S3_BUCKET_AWS_SECRET_ACCESS_KEY="${{secrets.EXTERNAL_S3_BUCKET_AWS_SECRET_ACCESS_KEY}}"
           if [ ${{ steps.otel-check.outputs.run_opentelemetry }} == "true" ]; then
-            export SYNAPSE_OTEL_INTEGRATION_TEST_EXPORTER="otlp"
+            # Set to 'otlp' to enable OpenTelemetry
```

Comment:
After inactivity or time it seems like the service catalog instance we set up with Jaeger stops - This doesn't prevent integration tests from running, however, it makes more noise when it prints that it cannot export the trace data.

Leaving this off for now and we can flip it on when want the data.